Formula/u/unitycatalog.rb

```diff
@@ -0,0 +1,45 @@
+class Unitycatalog < Formula
+  desc "Open, Multi-modal Catalog for Data & AI"
+  homepage "https://unitycatalog.io/"
+  url "https://github.com/unitycatalog/unitycatalog/archive/refs/tags/v0.3.0.tar.gz"
+  sha256 "fae708a22f1e38e19f754aca22925d66016a7efeaab680ce87c27496c75078d1"
+  license "Apache-2.0"
+
+  depends_on "openjdk@17"
```

Comment:
any issue tracking the jdk 21 or 24 support?

Reply:
Looks like it works with 21, so updated